signing_service/signing_service/tests/test_run.py

```diff
@@ -34,6 +34,7 @@ def setUp(self, unused_tcp_port_factory):
             SIGNING_SERVICE_PRIVATE_KEY,
             TEST_ETHEREUM_PRIVATE_KEY,
             SIGNING_SERVICE_DEFAULT_RECONNECT_ATTEMPTS,
+            Notifier(None, None, ['']),
```

Comment:
In tests, `SigningService` should be fed with `ConsoleNotifier` instance

Reply:
Done